aerogear-docs.adoc

```diff
@@ -0,0 +1,104 @@
+= Aerogear.org Website Renewal
+
+== Summary
+Everyone should be able to contribute to aerogear.org docs with ease and effectiveness, regardless of the scope of their proposed change.
```

Comment:
@finp I love this proposal! Really good job. 
We added some simple template to follow (mostly headers) that will help with verification and approving this proposal. See: https://github.com/aerogear/proposals/blob/master/template.md